docs/README.md

```diff
@@ -17,44 +11,87 @@ cd docs
 npm install
 ```
 
-This will download all dependencies, including Hugo.
+Then, start the server.
 
-#### Using Hugo manually (optional)
+```bash
+npm start
+```
 
-You can install Hugo manually if you want. Use the [instructions on the Hugo
-site](https://gohugo.io/getting-started/installing/) to download it.
+Open <http://localhost:1313> to preview the site.
 
-### Previewing the site
+### API documentation and Playground
 
-Once you have Hugo installed you can then start the
-developer preview site like so:
+The steps above won't include API documentation or the Playground by default.  You can
+download the latest content with the `download` script.
 
 ```bash
-npm start
+npm run download
 ```
 
-Open `http://localhost:1313` to preview the site. Any changes to the source
-content will automatically force an HTML re-render allowing you to preview your
-modifications in quasi real time.
+## Building the documentation
 
-If you want to debug the generated content, you can build the site and see the
-output in the `public/` folder:
+Run the `build` script to build the site. The output will be in the `public/` folder.
```

Comment:
I think it might be good to clarify here if when you build the site if it affects the site preview from npm start

Reply:
I wasn't aware - what does it affect?

Reply:
Build and start are not related. Start serves from an in-memory dev server and doesn't serve anything from the public folder.

Reply:
That said, there are some tasks that are only done in the build script for performance reasons. That's worth calling out.

Reply:
I think I'll omit for now, the purpose of this change is to add info on how to debug bad doc output.  Details on motivations for particular script decisions could be a separate change.